core/conn/jdbc_type2/src/main/java/org/apache/trafodion/jdbc/t2/SQLMXPooledConnectionManager.java

```diff
@@ -23,14 +23,14 @@
  * Filename : SQLMXPooledConnectionManager.java
  */
 
-package org.trafodion.jdbc.t2;
+package org.apache.trafodion.jdbc.t2;
 
 import java.sql.*;
 import javax.sql.*;
 import java.util.Properties;
 import java.util.LinkedList;
 import java.io.PrintWriter;
-import org.trafodion.jdbc.t2.*;
+import org.apache.trafodion.jdbc.t2.*;
```

Comment:
Just a minor question: Do we really need to import the content of our own package?